projects/jest-utils/tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
I changed the way the project `tsconfig.*.json` are setup. So they are more in line in what a new NX project would generate a lib.